cpp/tests/utilities/column_wrapper.cuh

```diff
@@ -604,9 +602,7 @@ struct column_wrapper {
     the_column.data = data.data().get();
     the_column.size = data.size();
     the_column.dtype = cudf::gdf_dtype_of<ColumnType>();
-    gdf_dtype_extra_info extra_info;
-    extra_info.time_unit = TIME_UNIT_NONE;
-    extra_info.category = nullptr;
+    gdf_dtype_extra_info extra_info{TIME_UNIT_NONE};
```

Comment:
Wouldn't it be better to zero initialize? For example, you missed initializing the `category` member here. 
```suggestion
    gdf_dtype_extra_info extra_info{};
```

Reply:
No need to add `category`, the remaining members not present in the braced-init-list are zero initialized

Reply:
I'm not sure if that's true. Both `gcc` and `clang` throw a warning if you don't provide initializers for the remaining members in the init list. Example: https://wandbox.org/permlink/dwPz7TMkJCG8MZmw 

Likewise I can't find anywhere in the standard where an incomplete brace initialization list guarantees that the remaining members are zero-initialized.

Reply:
From https://en.cppreference.com/w/cpp/language/aggregate_initialization
> If the number of initializer clauses is less than the number of members and bases (since C++17) or initializer list is completely empty, the remaining members and bases (since C++17) are initialized by their default member initializers, if provided in the class definition, and otherwise (since C++14) by empty lists, in accordance with the usual list-initialization rules (which performs value-initialization for non-class types and non-aggregate classes with default constructors, and aggregate initialization for aggregates). If a member of a reference type is one of these remaining members, the program is ill-formed.

Also, about the warnings https://stackoverflow.com/a/1539162

It's fine for me either way, would you like me to explicitly initialize all members?

Reply:
Aha, thanks for the relevant section of the standard. I was staring at that page for a while and must have glanced right over it. What you have is fine.